### PR TITLE
edit form created and populated from database

### PR DESCRIPTION
**Pull request by alecorra**

### CODE REVIEW

File: editFetchComand.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+$db = new PDO('mysql:host=192.168.20.20;dbname=portfolioProject', 'root', '');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$sql = 'SELECT `project_pic`, `url` FROM `portfolio` WHERE `id` = ?;';
+
+if (!empty($_GET['id']) && (is_numeric($_GET['id'])==true)) {
```
Comment (mporam) [inline editFetchComand.php]: you dont need the `==true` as is_numeric will return a boolean

File: editFetchComand.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    $id = $_GET['id'];
+    $query = $db->prepare($sql);
+    $query->execute([$id]);
+    $result=$query->fetch();
```
Comment (mporam) [inline editFetchComand.php]: `$result = $query..`

File: editForm.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <nav>
+        <img class="mypic" src="img/MyLogo.png">
+        <a class="portfolio" href="portfolio.php">PORTFOLIO</a>
+        <a class="home" href="index.html">HOME</a>
+    </nav>
+    <div>
+        <form method="post" action="editUpdateComand.php">
```
Comment (mporam) [inline editForm.php]: your going to need to pass the ID to editUpdateComand.php as well

Comment (mporam) [reply editForm.php]: `method='POST'`
please use capitals

File: editFetchComand.php (hunk 1 above)
Comment (alecorra) [reply editFetchComand.php]: removed

File: editFetchComand.php (hunk 2 above)
Comment (alecorra) [reply editFetchComand.php]: spaces added

File: editForm.php (hunk 3 above)
Comment (alecorra) [reply editForm.php]: hidden id added, post in capitol done

File: editFetchComand.php [hunk 4]
```diff
@@ -5,11 +5,11 @@
 
 $sql = 'SELECT `project_pic`, `url` FROM `portfolio` WHERE `id` = ?;';
 
-if (!empty($_GET['id']) && (is_numeric($_GET['id'])==true)) {
+if (!empty($_GET['id']) && (is_numeric($_GET['id']))) {
     $id = $_GET['id'];
     $query = $db->prepare($sql);
     $query->execute([$id]);
-    $result=$query->fetch();
+    $result = $query->fetch();
```
Comment (mporam) [inline editFetchComand.php]: you are not currently using $result. you need to check if result is true, then redirect, if its false, show an error

Comment (alecorra) [reply editFetchComand.php]: fixed

**Pull request merged**